vtr_flow/scripts/remove_3d_attributes.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script is intended to be used on RR graph XML files generated by VPR.
+# If the architecture is a 2D architecture, you can simply call this script
+# to remove all attributes related to 3D FPGA support (layer="0",
+# layer_low="0", layer_high="0"). In VPR, these values default to 0.
```

Comment:
I’d add a comment saying why this is useful.
1. Convert an older rr graph with layer set to 2D format as we now require either no layer or both layer_low and layer_high.
2. 2. It also modestly reduces the file size for 2D archs.